stripecardscan-example/src/main/res/layout/activity_card_scan_fragment_demo.xml

```diff
@@ -18,7 +18,7 @@
             android:layout_height="0dp"
             android:layout_marginTop="64dp"
             android:visibility="gone"
-            app:layout_constraintDimensionRatio="200:160"
+            app:layout_constraintDimensionRatio="1:1"
```

Comment:
is this change intended?

Reply:
yes. The demo squishes the fragment without this change.